app/views/posts/show.html.erb

```diff
@@ -10,4 +10,22 @@
     <p>comment #<%= index + 1 %></p>
     <p><%= user.name %>: <%= comment.text %></p>
   <% end %>
+
+  <%= form_with scope: :like, url: "/users/#{@user.id}/posts/#{@post.id}/likes", local: true do |form|%>
+    <p>
+      <%= form.check_box :like, onchange: "this.form.submit()" %>
+      <%= form.label :like%>
+    </p>
+  <% end %>
+  
+  <h4>Write your comment below</h4>
+  <%= form_with scope: :comment, url: "/users/#{@user.id}/posts/#{@post.id}/comments", local: true do |form|%>
+  <p>
+    <%= form.label :comment%> <br>
+    <%= form.text_area :text%>
+  </p>
```

Comment:
- [ ] This form should be found in views/comments/new.html.erb file which accepts input for creating comments as a best practice and for easy debugging.

Reply:
Thanks for the review @FrederickMih, I think creating `new.html.erb` file will be little bit redundant because there is no need rendering a new page for comments under posts